tests/manifest_test.py

```diff
@@ -21,6 +21,8 @@
 
 
 class TestFileLevelManifest:
+    _manifest_type = manifest._FileSerialization("unused")
```

Comment:
nit: it's not immediately clear what "unused" is

Reply:
I'll change